src/document_to_podcast/preprocessing/data_loaders.py

```diff
@@ -36,6 +40,55 @@ def load_docx(docx_file: str | UploadedFile) -> str | None:
         return None
 
 
+def load_file(file: str | UploadedFile) -> str | None:
+    """
+    Loads the content of a file and converts it to markdown.
+
+    Args:
+        file: Either a file path string or a Streamlit UploadedFile object
+
+    Returns:
+        The markdown text content if successful, None otherwise
+    """
+    if not hasattr(file, "name"):
+        logger.error("Invalid file object: missing 'name' attribute")
+        return None
```

Comment:
If the `file` is a `str` this will always return None. Lets drop the type hint and comment that `file` can be a file path? I dont think its useful